fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1647,6 +1656,7 @@ public boolean replayJournal(long toJournalId) throws JournalException {
         try {
             cursor = journal.read(replayedJournalId.get() + 1, toJournalId);
             hasLog = replayJournalInner(cursor, false);
+            cursor.close();
```

Comment:
this line is not needed, you will close it in finally block

Reply:
You're right. My mistake.